src/ui/RoomSettings.h

```diff
@@ -137,4 +141,5 @@ public slots:
     RoomInfo info_;
     int notifications_ = 0;
     int accessRules_   = 0;
+    std::vector<bool> hiddenEvents_;
```

Comment:
Probably makes more sense to make this a std::set or the Qt equivalent and check the count(event_type) value (or contains in newer C++/Qt).

Reply:
Right, that was a hack I forgot to change to something sensible. :see_no_evil:  Can I use `EventType`s in QML or should I just use strings?

Reply:
You can pass event type to qml, but if you want to handle it as anything but a number, you will need to convert it to the event type in the timelinemodel